runtime/compiler/env/VMJ9.cpp

```diff
@@ -594,7 +594,6 @@ TR_J9VMBase::get(J9JITConfig * jitConfig, J9VMThread * vmThread, VM_TYPE vmType)
       {
       // Check if this thread has cached the frontend inside
 
-#if defined(J9VM_INTERP_AOT_COMPILE_SUPPORT)
```

Comment:
The `else` block at line 636 should not be allowed if `J9VM_INTERP_AOT_COMPILE_SUPPORT` is not defined because in that block we created a frontend for JITSERVER wwith AOT support